### PR TITLE
Renaming "ergs" to gas in the docs.

### PR DESCRIPTION
**Pull request by krofax**
Updated "ergs" to gas, as regards VM, server, SDK(js only), and l1 ->  l2.

### CODE REVIEW

File: docs/api/api.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To specify additional fields, like the custom signature for custom accounts or t
 "factory_deps": ["0x..."]
```
Comment (AnastasiiaVashchuk) [inline docs/api/api.md:25]:
```suggestion
"factoryDeps": ["0x..."]
```

File: docs/api/api.md [hunk 2]
```diff
@@ -39,10 +39,10 @@ Instead of signing the RLP-encoded transaction, the user signs the following typ
 | txType                  | `uint256`   |
 | from                    | `uint256`   |
 | to                      | `uint256`   |
-| ergsLimit               | `uint256`   |
-| ergsPerPubdataByteLimit | `uint256`   |
-| maxFeePerErg            | `uint256 `  |
-| maxPriorityFeePerErg    | `uint256`   |
+| l2gasLimit              | `uint256`   |
+| l2PerPubdataByteLimit   | `uint256`   |
```
Comment (AnastasiiaVashchuk) [inline docs/api/api.md:43]:
```suggestion
| gasLimit              | `uint256`   |
| gasPerPubdataByteLimit   | `uint256`   |
```

File: docs/api/api.md [hunk 3]
```diff
@@ -76,11 +76,11 @@ Returns the fee for the transaction. The token in which the fee is calculated is
 
 ```json
 {
-  "ergs_limit": 100000000,
-  "ergs_price_limit": 10000,
+  "gass_limit": 100000000,
+  "gas_price_limit": 10000,
```
Comment (AnastasiiaVashchuk) [inline docs/api/api.md:80]:
```suggestion
  "gas_limit": 100000000,
  "max_fee_per_gas": 10000,
```

File: docs/api/api.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ Returns the fee for the transaction. The token in which the fee is calculated is
...
   "fee_token": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
-  "ergs_per_storage_limit": 100,
-  "ergs_per_pubdata_limit": 10
+  "gas_per_storage_limit": 100,
+  "gas_per_pubdata_limit": 10
```
Comment (AnastasiiaVashchuk) [inline docs/api/api.md:83]:
```suggestion
  "max_priority_fee_per_gas": 100,
  "gas_per_pubdata_limit": 10
```

File: docs/api/api.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ Returns the fee for the transaction. The token in which the fee is calculated is
 
 ```json
 {
-  "ergs_limit": 100000000,
-  "ergs_price_limit": 10000,
+  "gass_limit": 100000000,
+  "gas_price_limit": 10000,
   "fee_token": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
```
Comment (AnastasiiaVashchuk) [inline docs/api/api.md:81]:
```suggestion
```

File: docs/api/go/getting-started.md [hunk 6]
```diff
@@ -11,6 +11,12 @@ In this guide we will demonstrate how to:
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+⚠️ This section of the documentation is no longer current and needs to be updated to reflect the changes made to the system contract. We apologize for any inconvenience and assure you that a revised version will be available shortly.
```
Comment (AnastasiiaVashchuk) [inline docs/api/go/getting-started.md:16]:
```suggestion
⚠️ This section of the documentation is no longer current and needs to be updated to reflect the changes made to the system contracts. We apologize for any inconvenience and assure you that a revised version will be available shortly.
```

File: docs/api/js/accounts-l1-l2.md [hunk 7]
```diff
@@ -143,7 +143,7 @@ async finalizeWithdrawal(withdrawalHash: BytesLike, index: number = 0): Promise<
 
 ```ts
 async getBaseCost(params: {
-    ergsLimit: BigNumberish;
+    l2gasLimit: BigNumberish;
     calldataLength: number;
     gasPrice?: BigNumberish;
```
Comment (AnastasiiaVashchuk) [inline docs/api/js/accounts-l1-l2.md:148]:
```suggestion
     gasLimit: BigNumberish;
     gasPerPubdataByte?: BigNumberish;
     gasPrice?: BigNumberish;
```

File: docs/api/js/accounts-l1-l2.md [hunk 8]
```diff
@@ -179,7 +179,7 @@ async claimFailedDeposit(depositHash: BytesLike): Promise<ethers.ContractTransac
 async requestExecute(transaction: {
     contractAddress: Address;
     calldata: BytesLike;
-    ergsLimit: BigNumberish;
+    l2gasLimit: BigNumberish;
     factoryDeps?: ethers.BytesLike[];
     operatorTip?: BigNumberish;
     overrides?: ethers.CallOverrides;
```
Comment (AnastasiiaVashchuk) [inline docs/api/js/accounts-l1-l2.md:185]:
```suggestion
 l2GasLimit: BigNumberish;
 l2Value?: BigNumberish;
 factoryDeps?: ethers.BytesLike[];
 operatorTip?: BigNumberish;
 gasPerPubdataByte?: BigNumberish;
 refundRecipient?: Address;
 overrides?: ethers.PayableOverrides;
```

File: docs/api/js/accounts-l1-l2.md [hunk 9]
```diff
@@ -225,19 +225,19 @@ const abi = [
...
 const txCostPrice = await wallet.getBaseCost({
   gasPrice,
   calldataLength: ethers.utils.arrayify(calldata).length,
-  ergsLimit,
+  l2gasLimit,
 });
 
 console.log(`Executing the transaction will cost ${ethers.utils.formatEther(txCostPrice)} ETH`);
 
 const executeTx = await wallet.requestExecute({
```
Comment (AnastasiiaVashchuk) [inline docs/api/js/accounts-l1-l2.md:241]: please, update it so it reflects https://github.com/matter-labs/zksync-web-v2-docs/pull/274/files#r1098706461 these changes

Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:241]: you did not

**Comment by uF4No** [comment]
Can you rephrase the warning to: "The information in this section of the documentation is outdated and requires updating. We apologize for any inconvenience caused and assure you that a revised version will be made available soon."

**Comment by krofax** [comment]
> The information in this section of the documentation is outdated and requires updating. We apologize for any inconvenience caused and assure you that a revised version will be made available soon.

Is this for the tutorials or the SDKs? @uF4No

File: docs/api/js/accounts-l1-l2.md [hunk 10]
```diff
@@ -192,7 +195,7 @@ async requestExecute(transaction: {
...
 | transaction.calldata               | The calldata of the call transaction. It can be encoded the same way as in Ethereum.                                                                                                                                                                                                                                                                             |
-| transaction.ergsLimit              | The `ergsLimit` for the call.                                                                                                                                                                                                                                                                                                                                    |
+| transaction.l2gasLimit             | The `l2gasLimit` for the call.                                                                                                                                                                                                                                                                                                                                    |
```
Comment (StanislavBreadless) [inline docs/api/js/accounts-l1-l2.md:198]: It is better to be slightly more verbose here

```suggestion
| transaction.l2gasLimit             | The L2 gas limit for the call.                                                                                                                                                                                                                                                                                                                                    |
```

File: docs/api/js/getting-started.md [hunk 11]
```diff
@@ -3,6 +3,11 @@
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+Please note that in the new `0.13.0-beta.2` SDK we changed the API to rename ergs to gas, to avoid breaking changes in your integration, update all your project dependencies.
+
+:::
```
Comment (StanislavBreadless) [inline docs/api/js/getting-started.md:10]:
```suggestion
:::

```

File: docs/dev/developer-guides/aa.md [hunk 12]
```diff
@@ -3,6 +3,12 @@
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+Please note that in the new `0.13.0-beta.2` SDK we changed the API to rename ergs to gas, to avoid breaking changes in your integration, update all your project dependencies.
```
[the remark this answers is not in the document]
Comment (krofax) [reply docs/dev/developer-guides/aa.md:8]: Noted, will update once regenesis is done.

File: docs/api/js/accounts-l1-l2.md [hunk 13]
```diff
@@ -179,10 +179,13 @@ async claimFailedDeposit(depositHash: BytesLike): Promise<ethers.ContractTransac
 async requestExecute(transaction: {
     contractAddress: Address;
     calldata: BytesLike;
-    ergsLimit: BigNumberish;
-    factoryDeps?: ethers.BytesLike[];
-    operatorTip?: BigNumberish;
-    overrides?: ethers.CallOverrides;
+ l2GasLimit: BigNumberish;
```
[the remark this answers is not in the document]
Comment (AnastasiiaVashchuk) [reply docs/api/js/accounts-l1-l2.md:182]: @krofax  use `yarn lint:fix`

**Review by AnastasiiaVashchuk — DISMISSED** [review]
